### PR TITLE
Updated sqlcipher compile instructions for Fedora (Clarifies and closes  #2970)

### LINKED ISSUE #2970: Compiling sqlcipher(needed for qtox) in fedora
Here: 
There were instructions for fedora as : Note that sqlcipher is not included in Fedora(!).
This means that you have to compile sqlcipher yourself, otherwise compiling qTox will fail.
https://github.com/tux3/qTox/blob/master/INSTALL.md#sqlcipher

But at sqlcipher https://github.com/sqlcipher/sqlcipher the compiling instructions say : 

> Building SQLCipher is almost the same as compiling a regular version of SQLite with two small exceptions:  
> 1. You must define SQLITE_HAS_CODEC and SQLITE_TEMP_STORE=2 when building sqlcipher.
> 2.     You need to link against a OpenSSL's libcrypto

These instructions are not included at https://github.com/tux3/qTox/blob/master/INSTALL.md#sqlcipher So my question is which one to follow and why ? And which method is needed for compiling qtox in fedora ?

### PR DESCRIPTION
**Pull request by linux-modder**

### CODE REVIEW

**Comment by zetok** [comment]
@suknight does that clarify things for you?

**Comment by hethkar** [comment]
@zetok yes, thank you

File: INSTALL.md [hunk 1]
```diff
@@ -387,6 +388,24 @@ autoreconf -if
...
+
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
+
+Statically Linked:
+cd sqlpcipher
+$ ./configure --enable-tempstore=yes CFLAGS="-DSQLITE_HAS_CODEC" \
+    LDFLAGS="/opt/local/lib/libcrypto.a"
+$ make
+$ sudo make install
```
Comment (zetok) [inline INSTALL.md]: Could you remove all the unnecessary `$` ?

Also, put things that aren't commands outside of code block – each variant should be in a separate code block.

File: INSTALL.md [hunk 2]
```diff
@@ -380,6 +380,7 @@ sudo make install
 If you are not using Fedora, skip this section, and go directly to compiling [**toxcore**](#toxcore-compiling).
 
 ```bash
+This method Auto detects whether to statically or Dynamically  link per your system configs.
```
Comment (zetok) [inline INSTALL.md]: remove unneeded whitespace

File: INSTALL.md [hunk 3]
```diff
@@ -387,6 +388,24 @@ autoreconf -if
 make -j$(nproc)
 sudo make install
 cd ..
+
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
```
Comment (zetok) [inline INSTALL.md]: remove unneeded whitespace;

Why would `Conversely,` part be needed? It can start just from `If`

File: INSTALL.md [hunk 4]
```diff
@@ -387,6 +388,24 @@ autoreconf -if
...
+$ sudo make install
+cd ..
+
+Dynamically Linked:
+cd sqlcipher$ ./configure --enable-tempstore=yes CFLAGS="-DSQLITE_HAS_CODEC" \
+    LDFLAGS="-lcrypto"
+$ make
+$ sudo make install
+cd ..
 ````
```
Comment (zetok) [inline INSTALL.md]: while this isn't part of this changeset, it also could be changed to just    ```

File: INSTALL.md [hunk 5]
```diff
@@ -387,6 +388,24 @@ autoreconf -if
...
+
+Statically Linked:
+cd sqlpcipher
+$ ./configure --enable-tempstore=yes CFLAGS="-DSQLITE_HAS_CODEC" \
+    LDFLAGS="/opt/local/lib/libcrypto.a"
+$ make
+$ sudo make install
+cd ..
+
+Dynamically Linked:
```
Comment (zetok) [inline INSTALL.md]: Can be `#### Dynamically linked`

File: INSTALL.md [hunk 6]
```diff
@@ -387,6 +388,24 @@ autoreconf -if
 make -j$(nproc)
 sudo make install
 cd ..
+
+
+Conversely, if you wish to explictly statically  or dynamically link sqlcipher use:
+
+Statically Linked:
```
Comment (zetok) [inline INSTALL.md]: Can be `#### Statically linked`

**Comment by zetok** [comment]
Picked relevant commits in 6c0fb9a0f514d6b0ce82fe40db254f2cb31acfd6, thus closing.

**Pull request closed without merging**